pkg/scheduler/framework/plugins/helper/node_ports.go

```diff
@@ -14,16 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package predicates
+package helper
 
 import (
 	v1 "k8s.io/api/core/v1"
 	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 )
 
-// portsConflict check whether existingPorts and wantPorts conflict with each other
```

Comment:
we don't want to introduce more cleanup backlog, lets move this logic to predicates.go, just place it  inside ```PodFitsHostPortsPredicate```, there is no need for a separate function, and then the whole ```PodFitsHostPortsPredicate``` will move to nodeports plugin later.